src/main/java/frc/robot/commands/cannon/ShootPiece.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.cannon;
+
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.subsystems.Feeder;
+import frc.robot.subsystems.Shooter;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class ShootPiece extends ParallelRaceGroup {
+  /** Creates a new ShootPiece. */
+  public ShootPiece(Shooter shooter, Feeder feeder) {
+    addCommands(
+    new Shoot(shooter),
+    new SequentialCommandGroup(new StartFeeder(feeder), new WaitCommand(2))
```

Comment:
We probably want to wait just a bit before running the feeder, like .5 or less. Can the command do that?